### PR TITLE
Breaking change OpenSearch main project - Action movement

### PR DESCRIPTION
**Pull request by MitchellGale**
### Description
Breaking change by OpenSearch. New location for some functions, so libraries updated respectively.
https://github.com/opensearch-project/OpenSearch/pull/9082
 
### Issues Resolved
https://github.com/opensearch-project/sql/issues/1953
 
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass, including unit test, integration test and doctest
- [x] New functionality has been documented.
  - [x] New functionality has javadoc added
  - [x] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by matthewryanwells — APPROVED** [review]

File: opensearch/src/main/java/org/opensearch/sql/opensearch/planner/physical/MLCommonsOperatorActions.java [hunk 1]
```diff
@@ -220,7 +220,7 @@ protected MLPredictionOutput getMLPredictionResult(FunctionName functionName,
 
     return (MLPredictionOutput) machineLearningClient
             .trainAndPredict(mlinput)
-            .actionGet(30, TimeUnit.SECONDS);
+            .actionGet(30L, TimeUnit.SECONDS);
```
Comment (GumpacG) [inline opensearch/src/main/java/org/opensearch/sql/opensearch/planner/physical/MLCommonsOperatorActions.java:223]: Is this change required?

**Review by GumpacG — APPROVED** [review]

File: opensearch/src/main/java/org/opensearch/sql/opensearch/planner/physical/MLCommonsOperatorActions.java (hunk 1 above)
Comment (MitchellGale) [reply opensearch/src/main/java/org/opensearch/sql/opensearch/planner/physical/MLCommonsOperatorActions.java]: Nope, thanks. Removed both instances of the added `L`

**Comment by Yury-Fridlyand** [comment]
Why CI fails?

**Review by Yury-Fridlyand — APPROVED** [review]

**Comment by MitchellGale** [comment]
> Why CI fails?

Fixed now - was missing a few updates.

**Review by GumpacG — APPROVED** [review]
